internal/api/v1/handlers/instance.go

```diff
@@ -113,20 +114,3 @@ func (h *InstanceHandler) GetInstance(c *fiber.Ctx) error {
 		"error": "get instance not implemented yet",
 	})
 }
-
-// convertToInstances convierte DeleteInstance a Instance
-func convertToInstances(deleteInstances []infrastructure.DeleteInstance) []infrastructure.InstanceRequest {
```

Comment:
This is not being used currently. If we need it in the future we can just ignore the lint error for unused code for now.